sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -217,8 +221,32 @@ struct _pi_context : _pi_object {
     // we don't need a map with device as key.
     HostMemAllocContext = new USMAllocContext(
         std::unique_ptr<SystemMemory>(new USMHostMemoryAlloc(this)));
+
+    // If we have a single device/sub-device, construction is done.
+    if (NumDevices < 2)
```

Comment:
The check for "<2" looks a little weird, please check "==1" and assert that "!=0"

Reply:
Fixed